idd/Energy+.idd.in

```diff
@@ -60561,11 +60643,27 @@ AirLoopHVAC:UnitarySystem,
        \type choice
        \key UnitarySystemPerformance:Multispeed
        \note Enter the type of performance specification object used to describe the multispeed coil.
-  A27; \field Design Specification Multispeed Object Name
+  A27, \field Design Specification Multispeed Object Name
        \type object-list
        \object-list UnitarySystemPerformaceNames
        \note Enter the name of the performance specification object used to describe the multispeed coil.
-
+  A28, \field Allow UnitarySystem As Water Side Eonomizer
+       \type choice
+       \key Yes
+       \key No
+       \default No
+       \note If "yes" the water cooling coil is placed on the demand branch of condenser loop and used as a
+       \note waterside economizer. The unitarysystem is used a wrapper for water cooling coil.	   
+	   \note If "No", the coils are used as a regular unitary system. 
+  N27; \field Minimum Air To Water Temperature Offset
+       \note Water cooling coil will turn on as required when inlet air temperature is above
+       \note water temperature by amount of offset. To model a waterside economizer connect 
+       \note to condenser loop and increase offset as desired.
+       \type real
+       \units C
+       \minimum 0.0
+       \default 0.0	
+	   
```

Comment:
Added these two fields to be able to tell the AirLoopHVAC:UnitarySystem that it is used as a water-side economizer heat exchanger and apply the temperature offset when needed.